DEVELOPER_RESOURCES.md

```diff
@@ -0,0 +1,63 @@
+Kubernetes cluster. This enables users to delete those resources from
+the cluster without losing the information contained in those resources.
+
+## Communication
+Chat rooms and email addresses for KubeArchive communication.
+
+TBD
+
+## Dashboards
+TBD
+
+## Development Resources
+
+### Git Repository
+Git repository for the KubeArchive source code.
+
+* [kubearchive/kubearchive](https://github.com/kubearchive/kubearchive)
+
+### Go Programming Language
+Suggested resources for working with the Go programming language.
+
+* [Exercism learning track for Go](https://exercism.org/tracks/go)
+* [Golang By Example - Advanced Tutorial](https://golangbyexample.com/golang-comprehensive-tutorial/)
+* [Ardan Labs Ultimate Go Tour](https://tour.ardanlabs.com/tour/eng/list)
+* [Dave Cheney's Practical Go: Real Worl Advice Guide](https://dave.cheney.net/practical-go/presentations/qcon-china.html)
+
+### Add-ons
+Add-ons the KubeArchive uses.
+
+* [Helm - Package Managment for Kubernetes](https://helm.sh/docs/)
+
+### Observability
+Resources for working with OpenTelemetry
+
+* [OpenTelemetry Go documentation](https://opentelemetry.io/docs/languages/go/)
+* [OpenTelemetry repository for Go instrumentation](https://github.com/open-telemetry/opentelemetry-go-instrumentation)
+* [Honeycomb documentation for OpenTelemetry](https://docs.honeycomb.io/send-data/go/honeycomb-distribution/)
+
+### Kubernetes
+Kubernetes is an open-source container orchestration engine for automating
+the deployment, scaling, and management of containerized applications.
+This is a list of tutorials and eBooks for learning about Kubernetes.
+
+* [Learning Kubernetes Tutorial](https://www.redhat.com/en/topics/containers/learning-kubernetes-tutorial)
+* [Kubernetes Patterns eBook](https://developers.redhat.com/e-books/kubernetes-patterns)
+* [Kubernetes Operators eBook](https://developers.redhat.com/e-books/kubernetes-operators)
+
+### Red Hat OpenShift
+Red Hat OpenShift provides enterprise-ready enhancements to Kubernetes.
+This is a list of suggested Red Hat training courses for OpenShift developers.
+
+* [DO188 - Red Hat OpenShift Development I: Introduction to Containers with Podman](https://role.rhu.redhat.com/rol-rhu/app/catalog?q=DO188)
+* [DO288 - Red Hat OpenShift Developer II: Building and Deploying Cloud-native Applications](https://role.rhu.redhat.com/rol-rhu/app/catalog?q=DO288)
+* [DO244 - Developing Applications with Red Hat OpenShift Serverless and Knative](https://role.rhu.redhat.com/rol-rhu/app/catalog?q=DO244)
+* [DO328 - Building Resilient Microservices with Istio and Red Hat OpenShift Service Mesh](https://role.rhu.redhat.com/rol-rhu/app/catalog?q=DO328)
+* [DO378 - Red Hat Cloud-native Microservices Development with Quarkus](https://role.rhu.redhat.com/rol-rhu/app/catalog?q=DO378)
+* [AD482 - Developing Event-driven Applications with Apache Kafka and Red Hat AMQ Streams](https://role.rhu.redhat.com/rol-rhu/app/catalog?q=AD482)
```

Comment:
I would rather just link to the software pages. Linking here paid courses and register-available books from Red Hat does not feel right to me, even if we are working for Red Hat. I think we should aim to maintain this project Red Hat agnostic.

```suggestion
* [Kubernetes - open-source container orchestration](https://kubernetes.io/)
* [OpenTelemetry](https://opentelemetry.io/)

```